airframe-rx-html/.js/src/test/scala/wvlet/airframe/http/rx/html/NestedElementTest.scala

```diff
@@ -34,9 +34,7 @@ object NestedElementTest extends AirSpec {
     div(
       id -> "nested",
       // Seq[RxElement]
-      lst.toSeq.map { i => new MyElement(s"button ${i}") },
-      // Iterable[RxElement]
-      lst.toIterable.map { i => new MyElement(s"button ${i}") }
+      lst.toSeq.map { i => new MyElement(s"button ${i}") }
```

Comment:
`toIterable` is deprecated, but there is no direct counterpart, so just deleted.